website/shared/editlinks.js

```diff
@@ -1,6 +1,10 @@
 const editLinksModule = (function(window) {
     // Function definitions
   
+    fetch('../../shared/rightmenu.js').then(response => response.text()).then(data => {
+        eval(data);
+    })
+
```

Comment:
You don't really do that, because that's a HTTP request to load a JS, why not just include it:
Try `<script src="/website/shared/rightmenu.js"></script>` in this file before line 183 https://github.com/devonfw/devonfw.github.io/blob/84036f9413d8102142069335b5c87056b8a481f4/asciidoctor-backend/erb/html5/document.html.erb#L183